src/Land/Model/SoilHeatParameterizations.jl

```diff
@@ -0,0 +1,196 @@
+"""
+    SoilHeatParameterizations
+
+Functions for volumetric heat capacity, internal energy as function of temperature,
+saturated thermal conductivity, thermal conductivty, the Kersten number, and relative
+saturation are included.
+"""
+module SoilHeatParameterizations
+
+using DocStringExtensions
+
+export volumetric_heat_capacity,
+    internal_energy,
+    saturated_thermal_conductivity,
+    thermal_conductivity,
+    relative_saturation,
+    kersten_number,
+    internal_energy_liquid_water,
+    temperature_from_I
+
+
+"""
+    function temperature_from_I(
+        T_ref::FT,
+        I::FT,
+        θ_i::FT,
+        ρ_i::FT,
+        LH_f_0::FT,
+        cs::FT
+    ) where {FT}
+
+Computes the temperature given I and `θ_i`.
+"""
+function temperature_from_I(
+    T_ref::FT,
+    I::FT,
+    θ_i::FT,
+    ρ_i::FT,
+    LH_f0::FT,
+    cs::FT,
+) where {FT}
+    T = T_ref + (I + θ_i * ρ_i * LH_f0) / cs
+    return T
+end
+
+"""
+    volumetric_heat_capacity(
+        θ_l::FT,
+        θ_i::FT,
+        c_ds::FT,
+        cp_l::FT,
+        cp_i::FT
+    ) where {FT}
+
+Compute the expression for volumetric heat capacity.
+"""
+function volumetric_heat_capacity(
+    θ_l::FT,
+    θ_i::FT,
+    c_ds::FT,
+    cp_l::FT,
+    cp_i::FT,
+) where {FT}
+    c_s = c_ds + θ_l * cp_l + θ_i * cp_i
+    return c_s
+end
+
+"""
+    internal_energy(
+        θ_i::FT,
+        c_s::FT,
+        T::FT,
+        T_ref::FT,
+        ρ_i::FT,
+        LH_f_0::FT
+    ) where {FT}
+Compute the expression for internal energy.
+"""
+function internal_energy(
```

Comment:
could we also note in the docs that this is volumetric internal energy

Reply:
done